StepDefinitions/LanguageStep.cs

```diff
@@ -0,0 +1,126 @@
+using Marsqa1Specflow.Pages;
+using Marsqa1Specflow.Utilities;
+using NUnit.Framework;
+using OpenQA.Selenium;
+
+namespace Marsqa1Specflow.StepDefinitions
+{
+    [Binding]
+    public class LanguageStep
+    {
+        private IWebDriver driver;
+
+        public LanguageStep(IWebDriver driver)
+        {
+            this.driver = driver;
+        }
+
+        [Given(@"User at Profile Page")]
+        public void NavigateProfilePage()
+        {
+            HomePage homepage = new HomePage(driver);
+            ProfilePage profilepage = new ProfilePage(driver);
```

Comment:
HomePage homepage = new HomePage(driver);
            ProfilePage profilepage = new ProfilePage(driver);

you can move this to the constructor and create a instance variable so that you can use in the other steps